packages/story-editor/src/components/panels/design/pageBackground/pageBackground.js

```diff
@@ -228,6 +228,7 @@ function PageBackgroundPanel({ selectedElements, pushUpdate }) {
             label={__('Background color', 'web-stories')}
             allowsSavedColors
             allowsOpacity={false}
+            pickerHasEyedropper={false}
```

Comment:
Hm, it has eyedropper currently though, is this removed purposefully? Choosing a color from canvas should still be available for the background color, no?

Reply:
Oh no, I was just experimenting with the new options before I added the storybook-stories. Removed in ad78121ca8.